### PR TITLE
Fix ElasticSearch module for python3

### PR DESCRIPTION
**Pull request by gpoulin**
Also activate ElasticSearch test on Travis

### CODE REVIEW

File: test/esindex_test.py [hunk 1]
```diff
@@ -36,6 +36,11 @@
 import os
 import unittest
 
+try:
+    from unittest import SkipTest
+except ImportError:
+    from unittest2 import SkipTest
```
Comment (Tarrasch) [inline test/esindex_test.py]: I see this pattern being copied at many places. Can we have a `luigi.unittest` that abstracts over this?

Comment (Tarrasch) [inline test/esindex_test.py]: Or `test.unittest` I suppose.

**Comment by Tarrasch** [comment]
LGTM

**Comment by erikbern** [comment]
Nice, LGTM. Too bad we don't run coveralls right now since coverage should go up

**Comment by gpoulin** [comment]
@erikbern We probably can add back coveralls. The issue with Travis wasn't with coveralls but with `server_test.py`.

**Comment by Tarrasch** [comment]
I'm fine with re-adding it coveralls. #776 For reference.

**Comment by gpoulin** [comment]
@Tarrasch hadoop tests are failing when trying to import a module in the test folder (https://travis-ci.org/spotify/luigi/jobs/52639913#L199). This is due to the fact that the module is not importable from the hadoop process (https://travis-ci.org/spotify/luigi/jobs/52639913#L362). I'll keep the `try ... except ...` in minicluster to prevent this issue.

**Comment by Tarrasch** [comment]
Cool. I'm fine with merging! :)

**Comment by Tarrasch** [comment]
Wait. Can you also please add link back to your comment from the minicluster.py file then? It's kind of surprising that you import it differently only at one place. ^^

https://github.com/spotify/luigi/blob/d4fc6b7417a6fb5122f8514837470b2d919f280b/test/minicluster.py#L26-L29

**Comment by erikbern** [comment]
Looks like test coverage is up to 81% now – nice!

https://coveralls.io/r/spotify/luigi?branch=master

**Comment by gpoulin** [comment]
@Tarrasch I added a comment in the code and reworded my commit.

**Comment by Tarrasch** [comment]
Merge conflict ...

**Comment by gpoulin** [comment]
unittest2 seems to break python3 (https://travis-ci.org/spotify/luigi/jobs/52750481#L1115). I'll try to import unittest2 only for python2.6 which is the only version that actually need it.

**Comment by Tarrasch** [comment]
Is this ready to be merged?

**Comment by gpoulin** [comment]
@Tarrasch yeah!

**Comment by Tarrasch** [comment]
Thanks!

**Pull request merged**